src/components/PivotTable/Droppable/Droppable.tsx

```diff
@@ -144,15 +142,15 @@ export function Droppable<T>(props: DroppableProps<T>) {
               key={key as string}
               type={accept}
               name={key}
-              value={keyMapping.get(key) ? keyMapping.get(key).keyName : (key as string)}
+              value={keyMap.get(key) ? keyMap.get(key).keyName : (key as string)}
               origin={name}
               onDragEnd={() => deleteByKey(key)}
               onKeyNav={onKeyNav}
             />
           )
         }
       }),
-    [keyState, filter, accept, keyMapping, name, onKeyNav, deleteByKey]
+    [keyState, filter, accept, keyMap, name, onKeyNav, deleteByKey]
```

Comment:
![Screenshot from 2024-10-07 13-09-53](https://github.com/user-attachments/assets/12b60d80-2487-4571-bd83-c5945511042d)
Can you take a look at this?